HouseRules_Essentials/README.md

```diff
@@ -1,35 +1,244 @@
+    - Positive numbers increase damage, negative decrease.
+    - Config accepts Dictionary `{ "AbilityName1", int, "AbilityName2", int }`
+    ###### _Example JSON config for this rule_
+    ```json
+    {
+        "Rule": "AbilityDamageAdjustedRule",
+        "Config": { 
+          "Zap": 1,
+          "Whirlwind": 1
+        }
+    },
+    ```
 - **AbilityActionCostAdjustedRule**: Ability Action Cost is adjusted
-- **ActionPointsAdjustedRule**: Action points are adjusted
+    - Set whether abilities cost AP to cast or not.
+    - `true` means the ability has a cost to cast, `false` means that it doesn't.
+    - Config accepts Dictionary e.g. `{ "AbilityName1", bool, "AbilityName2", bool }`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "AbilityActionCostAdjustedRule",
+    "Config": {
+      "Zap": false,
+      "StrengthenCourage": false
+    }
+  },
+    ```
+- _**ActionPointsAdjustedRule**: Action points are adjusted_
+    - No longer needed? PieceConfig can do this too.
 - **CardEnergyFromAttackMultipliedRule**: Card energy from attack is multiplied
+    - Multiply how quickly the mana bar fills up when you attack enemies.
+    - Config accepts float e.g. `1.3`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "CardEnergyFromAttackMultipliedRule",
+    "Config": 2.0
+  },
+    ```
 - **CardEnergyFromRecyclingMultipliedRule**: Card energy from recycling is multiplied
+    - Config accepts float e.g `1.4`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "CardEnergyFromRecyclingMultipliedRule",
+    "Config": 2.0
+  },
+    ```
 - **CardLimitModifiedRule**: Card limit is modified
+    - Change the size of the player's card hand from the default 10/11
+    - Config accepts Int e.g `15`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "CardLimitModifiedRule",
+    "Config": 20
+  },
+    ```
 - **CardSellValueMultipliedRule**: Card sell values are multiplied
+    - Increase card sale values in the shop. 
+    - Config accepts float e.g `2.5`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "CardSellValueMultipliedRule",
+    "Config": 2.0
+  },
+    ```
 - **EnemyAttackScaledRule**: Enemy attack damage is scaled
+    - Config accepts float e.g `0.85`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "EnemyAttackScaledRule",
+    "Config": 0.85
+  },
+    ```
 - **EnemyDoorOpeningDisabledRule**: Enemy door opening ability disabled
+    - Config accepts bool e.g `true`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "EnemyDoorOpeningDisabledRule",
+    "Config": true
+  },
+    ```
 - **EnemyHealthScaledRule**: Enemy health is scaled
+    - Config accepts float e.g `0.85`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "EnemyHealthScaledRule",
```

Comment:
Another side note:

While the actual defining class `EnemyHealthScaledRule.cs` has the `Rule` suffix, we could alter the logic of the JSON reader/writer so as to not care about the `Rule` suffix.  Not pushing for one way or another, just mentioning as that thought was sparked.

Reply:
I agree that having a 'Rule' suffix is pointles here. It would be nicer not to have it.